assets/js/components/dashboard-sharing/ModuleRecoveryAlert/index.js

```diff
@@ -72,198 +61,48 @@ export default function ModuleRecoveryAlert() {
-		description = sprintf(
-			/* translators: %s: module name. */
-			__(
-				'%s data was previously shared with other users on the site by another admin who no longer has access. To restore access, you may recover the module as the new owner.',
-				'google-site-kit'
-			),
-			recoverableModules[ userAccessibleModules[ 0 ] ].name
-		);
-		children = (
-			<Fragment>
-				<p className="googlesitekit-publisher-win__desc">
-					{ __(
-						'By recovering the module, you will restore access for other users by sharing access via your Google account. This does not make any changes to external services and can be managed at any time via the dashboard sharing settings.',
-						'google-site-kit'
-					) }
-				</p>
-				{ Object.keys( recoveryErrors ).length > 0 && (
-					<Errors recoveryErrors={ recoveryErrors } />
-				) }
-			</Fragment>
-		);
-		onCTAClick = handleRecoverModules;
-	} else {
-		description = __(
-			'The data for the following modules was previously shared with other users on the site by another admin who no longer has access. To restore access, you may recover the module as the new owner.',
-			'google-site-kit'
-		);
-		children = (
-			<Fragment>
-				{ userAccessibleModules.map( ( slug ) => (
-					<div key={ slug }>
-						<Checkbox
-							checked={ checkboxes[ slug ] }
-							name="module-recovery-alert-checkbox"
-							id={ `module-recovery-alert-checkbox-${ slug }` }
-							onChange={ () => updateCheckboxes( slug ) }
-							value={ slug }
-							disabled={ recoveringModules }
-						>
-							{ recoverableModules[ slug ].name }
-						</Checkbox>
-					</div>
-				) ) }
-				<p className="googlesitekit-publisher-win__desc">
-					{ __(
-						'By recovering the selected modules, you will restore access for other users by sharing access via your Google account. This does not make any changes to external services and can be managed at any time via the dashboard sharing settings.',
-						'google-site-kit'
-					) }
-				</p>
-				{ Object.keys( recoveryErrors ).length > 0 && (
-					<Errors recoveryErrors={ recoveryErrors } />
 				) }
-			</Fragment>
-		);
-		onCTAClick = handleRecoverModules;
-	}
-
-	return (
-		<BannerNotification
-			id="module-recovery-alert"
-			title={ __(
-				'Dashboard data for some services has been interrupted',
-				'google-site-kit'
-			) }
-			onCTAClick={ onCTAClick }
-			ctaLabel={ onCTAClick ? __( 'Recover', 'google-site-kit' ) : null }
-			ctaLink={ onCTAClick ? '#' : null }
-			description={ description }
-			learnMoreURL={ documentationURL }
-			learnMoreLabel={ __( 'Learn more', 'google-site-kit' ) }
-			isDismissible={ isDismissible }
-			dismiss={ __( 'Remind me later', 'google-site-kit' ) }
-			dismissExpires={ DAY_IN_SECONDS }
-		>
-			{ children }
-		</BannerNotification>
+				description={
+					<Description
+						id={ id }
+						recoverableModules={ recoverableModules }
+						userAccessibleModules={ userAccessibleModules }
```

Comment:
Please also use the same naming here as well
```suggestion
						userRecoverableModules={ userRecoverableModules }
```